awx/ui/client/test/e2e/tests/test-auditor-read-only-forms.js

```diff
@@ -0,0 +1,444 @@
+                })
+                .then(({ data }) => {
+                    adminMachineCredential.credential_type = data.id;
+
+                    return $http.post('/api/v2/credentials/', adminMachineCredential);
+                })
+                .then(({ data }) => {
+                    adminMachineCredential = data;
+                    adminJobTemplate.credential = data.id;
+
+                    return $http.post('/api/v2/users/', adminUser);
+                })
+                .then(({ data }) => {
+                    adminUser = data;
+
+                    return $http.post('/api/v2/notification_templates/', adminNotificationTemplate);
+                })
+                .then(({ data }) => {
+                    adminNotificationTemplate = data;
+
+                    return $http.post('/api/v2/inventory_scripts/', adminInventoryScript);
+                })
+                .then(({ data }) => {
+                    adminInventoryScript = data;
+
+                    return $http.post('/api/v2/projects/', adminProject);
+                })
+                .then(({ data }) => {
+                    adminProject = data;
+
+                    return $http.post('/api/v2/inventories/', adminSmartInventory);
+                })
+                .then(({ data }) => {
+                    adminSmartInventory = data;
+
+                    return $http.post('/api/v2/inventories/', adminStandardInventory);
+                })
+                .then(({ data }) => {
+                    adminStandardInventory = data;
+                    adminJobTemplate.inventory = data.id;
+
+                    return $http.post('/api/v2/job_templates/', adminJobTemplate);
+                })
+                .then(({ data }) => {
+                    adminJobTemplate = data;
+
+                    return $http.post('/api/v2/users/', auditor);
+                })
+                .then(({ data }) => {
+                    auditor = data;
+
+                    return {
+                        adminJobTemplate,
+                        adminAWSCredential,
+                        adminMachineCredential,
+                        adminOrganization,
+                        adminInventoryScript,
+                        adminNotificationTemplate,
+                        adminProject,
+                        adminSmartInventory,
+                        adminStandardInventory,
+                        adminTeam,
+                        adminUser,
+                        auditor
+                    };
+                });
+        },
+        ({ adminJobTemplate,
+           adminAWSCredential,
+           adminMachineCredential,
+           adminOrganization,
+           adminInventoryScript,
+           adminNotificationTemplate,
+           adminProject,
+           adminSmartInventory,
+           adminStandardInventory,
+           adminTeam,
+           adminUser,
+           auditor }) => {
+            store.created = {
```

Comment:
Since we're populating the 'created' field in one shot after doing the setup, we probably don't need to destructure the callback args here
```
...
},
 created => {
    store.created = created;
}
```